dallinger/prolific.py

```diff
@@ -20,11 +20,11 @@ class ProlificServiceException(Exception):
     """Some error from Prolific"""
 
 
-class ProlificServiceNoSuchProject(Exception):
+class ProlificServiceNoSuchProjectException(Exception):
```

Comment:
I liked it better the first way (no "Exception" in the name), but completely a cosmetic preference. I see I managed to name exceptions both with and without [...]Exception in `dallinger.mturk.py` 😄

Reply:
I agree with you.  It's unnecessarily wordy.  But I thought to add it because it's a more explicit way of naming the class, and some people prefer that.

You know what, I'll throw caution to the wind.  I'll revert this change.  Thanks.